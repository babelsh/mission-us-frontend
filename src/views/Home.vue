<template>
  <div class="grey lighten-3">  
    <v-card>
      <v-container fluid grid-list-lg>
        <v-layout row wrap>
          <v-flex xs12>
            <v-card color="blue darken-1" class="white--text" >
              <v-layout>            
                <v-flex xs7>
                  <v-card-title primary-title>
                    <div>
                      <div class="headline">Descobrindo seu filho</div>
                      <div>O que você precisa saber sobre seu filho?</div>
                    </div>
                  </v-card-title>
                </v-flex>
                <v-flex xs5>
                  <div class="text-xs-center">
                    <v-btn fab dark small color="blue " >
                      <div class="headline">1</div>
                    </v-btn>
                    <v-btn fab dark small color="blue " >
                      <div class="headline">2</div>
                    </v-btn>
                    <v-btn fab dark small color="blue " >
                      <div class="headline">3</div>                       
                    </v-btn>
                  </div>
                  <div class="text-xs-center">em progresso</div>
                </v-flex>
              </v-layout>
              <v-divider light></v-divider>               
            </v-card>
          </v-flex>    
        </v-layout>
          
        <v-timeline align-top dense>
          <v-timeline-item color="blue">
            <v-layout pt-3>            
              <v-flex>
                <v-avatar>
                  <v-img
                    src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQuWMekAs_Ldlhug3VGPGqv-0qSstNzs2SgxDCEMl79OAMMKbca"
                  />
                </v-avatar>
               <strong>A Criança e seu Desenvolvimento</strong>
                <div class="caption">
                  Ambiente, afeto, rotina: tudo isso exerce influência na evolução da criança. E os cuidados devem começar ainda na gravidez.
                </div>
              </v-flex>
            </v-layout>
            <v-btn block color="blue" dark>iniciar</v-btn>
          </v-timeline-item>
  
          <v-timeline-item color="grey">
            <v-layout wrap pt-3>
              <v-flex>
                <v-avatar>
                  <v-img
                    src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTNnhr8k4XVKfUomGttYdMbEVQafTcRkdyFMvI1YspFltIpoCCv"
                  />
                </v-avatar>
                <strong>Afeto é fundamental</strong>
                <div class="caption mb-2">
                  A maior parte das pessoas não relaciona as ligações afetivas ao desenvolvimento infantil.
                  Apesar de reconhecer que o carinho dos adultos é importante e influencia o temperamento da criança,
                  apenas 12% da população] considera o afeto importante para o desenvolvimento.
                </div>
                 <v-btn block color="grey lighten-1" dark>bloqueado</v-btn>
              </v-flex>
            </v-layout>
          </v-timeline-item>
  
          <v-timeline-item color="grey">
            <v-layout wrap pt-3>
              <v-flex>
                <v-avatar>
                  <v-img
                    src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS_Sl_A75vqKCouBWwshmzdokwnBILnYctL-C4TCw-9XPCocFMyBg"
                  />
                </v-avatar>
                <strong>&nbsp; &nbsp; Aprendizado a toda hora e em todo lugar</strong>
                <div class="caption mb-2">ara muitos brasileiros, o período mais importante da infância se inicia juntamente com a formação das lembranças que a criança levará consigo até a fase adulta, ou seja, entre os 2 e os 4 anos. E a maioria da população não considera que atividades como andar, correr, falar, entre outras, também façam parte do aprendizado cognitivo.</div>                        
                <v-btn block color="grey lighten-1" dark>bloqueado</v-btn>
              </v-flex>
            </v-layout>
          </v-timeline-item>  
        </v-timeline> 
      </v-container>

      <v-bottom-nav absolute color="transparent">
        <v-btn flat color="primary">Progresso</v-btn>
        <v-btn flat color="primary">Dúvidas</v-btn>
        <v-btn flat color="primary">Profissionais</v-btn>
      </v-bottom-nav>
    </v-card>
  </div>
</template>

<script>
import ActivityCard from '@/components/ActivityCard.vue';
import Trail from '@/components/Trail.vue';

export default {
  name: 'home',
  components: {
    ActivityCard,
    Trail,
  },
  data() {
    return {
      
    };
  },
  methods: {
    goToDescobrindoSeuFilho() {
      console.log('opaa');
      this.$router.push({ name: 'descobrindo' })
    },
  },
};
</script>
