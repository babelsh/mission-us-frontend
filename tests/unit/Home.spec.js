import { mount } from '@vue/test-utils';
import Home from '../../src/views/Home.vue';

describe('Home', () => {
  test('Home is a Vue instance', () => {
    const wrapper = mount(Home);
    expect(wrapper.isVueInstance()).toBeTruthy();
  });
});
