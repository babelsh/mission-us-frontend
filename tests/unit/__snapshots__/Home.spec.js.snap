// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the component properly 1`] = `
<div class="container fluid">
  <div class="container fluid">
    <div>
      <div class="layout text-xs-center">
        <div class="flex"><img src="../images/logo.png" height="150px"></div>
      </div>
    </div>
    <p class="title">Trilha 1</p>
    <p class="body-2 grey--text text-truncate">
      Esta é a primeira trilha de todas. Vamos lá?
    </p>
    <div class="layout">
      <div class="flex xs12"><button type="button" class="primary v-btn theme--light">
          <div class="v-btn__content">SABER MAIS</div>
        </button></div>
    </div>
  </div>
  <div class="container fluid grid-list-md ma-0 pa-0">
    <div class="layout row wrap">
      <div class="flex xs12 sm6 md4 lg3 xl3">
        <div class="container pa-0">
          <div class="v-card theme--light">
            <div class="v-responsive v-image" style="height: 200px;">
              <div class="v-image__image v-image__image--preload v-image__image--cover" style="background-position: center center;" name="fade-transition" mode="in-out"></div>
              <div class="v-responsive__content"></div>
            </div>
            <div class="v-card__title no-padding-bottom">
              <div class="text-truncate">
                <p class="title text-truncate">
                  Jogar futebol
                </p>
                <p class="body-2 grey--text text-truncate">
                  Jogar futebol com os vizinhos no play
                </p>
              </div>
            </div>
            <div class="v-card__actions no-padding-top"><button type="button" class="v-btn v-btn--flat theme--light">
                <div class="v-btn__content">
                  Saber mais
                </div>
              </button>
              <div class="spacer"></div> <button type="button" class="v-btn theme--light success">
                <div class="v-btn__content">
                  Finalizada
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
      <div class="flex xs12 sm6 md4 lg3 xl3">
        <div class="container pa-0">
          <div class="v-card theme--light">
            <div class="v-responsive v-image" style="height: 200px;">
              <div class="v-image__image v-image__image--preload v-image__image--cover" style="background-position: center center;" name="fade-transition" mode="in-out"></div>
              <div class="v-responsive__content"></div>
            </div>
            <div class="v-card__title no-padding-bottom">
              <div class="text-truncate">
                <p class="title text-truncate">
                  Estudar história
                </p>
                <p class="body-2 grey--text text-truncate">
                  Utilizar de mídias audiovisuais para estudo de história
                </p>
              </div>
            </div>
            <div class="v-card__actions no-padding-top"><button type="button" class="v-btn v-btn--flat theme--light">
                <div class="v-btn__content">
                  Saber mais
                </div>
              </button>
              <div class="spacer"></div> <button type="button" class="v-btn theme--light success">
                <div class="v-btn__content">
                  Finalizada
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
      <div class="flex xs12 sm6 md4 lg3 xl3">
        <div class="container pa-0">
          <div class="v-card theme--light">
            <div class="v-responsive v-image" style="height: 200px;">
              <div class="v-image__image v-image__image--preload v-image__image--cover" style="background-position: center center;" name="fade-transition" mode="in-out"></div>
              <div class="v-responsive__content"></div>
            </div>
            <div class="v-card__title no-padding-bottom">
              <div class="text-truncate">
                <p class="title text-truncate">
                  Jogar videogame
                </p>
                <p class="body-2 grey--text text-truncate">
                  Utilizar de jogos sérios para incentivar o ensino de matemática
                </p>
              </div>
            </div>
            <div class="v-card__actions no-padding-top"><button type="button" class="v-btn v-btn--flat theme--light">
                <div class="v-btn__content">
                  Saber mais
                </div>
              </button>
              <div class="spacer"></div> <button type="button" class="v-btn theme--light secondary">
                <div class="v-btn__content">
                  Feito
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
      <div class="flex xs12 sm6 md4 lg3 xl3">
        <div class="container pa-0">
          <div class="v-card theme--light">
            <div class="v-responsive v-image" style="height: 200px;">
              <div class="v-image__image v-image__image--preload v-image__image--cover" style="background-position: center center;" name="fade-transition" mode="in-out"></div>
              <div class="v-responsive__content"></div>
            </div>
            <div class="v-card__title no-padding-bottom">
              <div class="text-truncate">
                <p class="title text-truncate">
                  Celebração
                </p>
                <p class="body-2 grey--text text-truncate">
                  Celebrar o desenvolvimento da trilha
                </p>
              </div>
            </div>
            <div class="v-card__actions no-padding-top"><button type="button" class="v-btn v-btn--flat theme--light">
                <div class="v-btn__content">
                  Saber mais
                </div>
              </button>
              <div class="spacer"></div> <button type="button" class="v-btn theme--light secondary">
                <div class="v-btn__content">
                  Feito
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
